moveit_ros/occupancy_map_monitor/CMakeLists.txt

```diff
@@ -1,10 +1,18 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 3.10.2)
 project(moveit_ros_occupancy_map_monitor)
 set(MOVEIT_LIB_NAME ${PROJECT_NAME})
 
-set(CMAKE_CXX_STANDARD 14)
+# Default to C++14
```

Comment:
I think we should keep the original cmake flags and warnings and modify them globally if necessary

Reply:
Changed back to the original setting.